cmd/tailscale/cli/network-lock.go

```diff
@@ -151,13 +151,15 @@ func runNetworkLockInit(ctx context.Context, args []string) error {
 		return nil
 	}
 
-	fmt.Printf("%d disablement secrets have been generated and are printed below. Take note of them now, they WILL NOT be shown again.\n", nlInitArgs.numDisablements)
+	var disablementSecretsResp strings.Builder
```

Comment:
Perhaps `successMsg` would be a better name for this variable, describing the reason we have it as opposed to its content?

Reply:
Changed!